manage_breast_screening/notifications/management/commands/create_appointments.py

```diff
@@ -33,13 +33,12 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         try:
-            container_client = BlobStorage().find_or_create_container(
-                os.getenv("BLOB_CONTAINER_NAME")
-            )
-            for blob in container_client.list_blobs(
+            logger.info("Create Appointments Command started")
```

Comment:
looks like you've removed the `container_client` here? we're no longer getting it from the `self.container_client` method